### PR TITLE
EIN field in the general settings

### PR DESCRIPTION
**Pull request by ronaldblanco**
EIN field in the general settings was created to contributed to the solution of the issue #119.

The changes were tested locally and was also run the eslint check for client an server without any error.

### CODE REVIEW

File: client/modules/settings/components/General.js [hunk 1]
```diff
@@ -68,14 +68,22 @@ const General = () =>
...
             type="text"
             required
           />
         </Col>
+        <Col lg={3}>
+          <RFFieldGroup
+            name="eIN"
+            label="EIN"
+            type="text"
+            required
```
Comment (kenjiO) [inline client/modules/settings/components/General.js]: I'm not sure how we are going to use EIN yet so can you delete the required prop here.  Or if you think it should be required can you add code for the server that validates EIN is present before saving to the database.  Thanks!

File: client/modules/settings/components/General.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ const General = () =>
             rows="5"
           />
         </Col>
-        <Col lg={3}>
+        <Col lg={1}>
```
Comment (jspaine) [inline client/modules/settings/components/General.js]: Isn't it a bit small and weird looking having 1 and 2 as the width of these? Why not keep them all 3 and move the address to the end if it's in the way?

File: client/modules/settings/components/General.js [hunk 3]
```diff
@@ -68,14 +68,22 @@ const General = () =>
...
           <RFFieldGroup
             name="moneyUnit"
             label="Money Unit"
             type="text"
             required
           />
         </Col>
+        <Col lg={3}>
+          <RFFieldGroup
+            name="eIN"
```
Comment (jspaine) [inline client/modules/settings/components/General.js]: Why eIN and not EIN?

File: client/modules/settings/components/General.js (hunk 1 above)
Comment (jspaine) [reply client/modules/settings/components/General.js]: It's for the donation receipt, but yeah, I think it shouldn't be required.

**Comment by ronaldblanco** [comment]
Hello, I did follow the suggestions; take a look now.

Thanks.

**Comment by jspaine** [comment]
Thanks, looks good.

**Pull request merged**